public/js/script.js

```diff
@@ -99,6 +105,29 @@ function mountPlots(screenData, sampleData, featureNames) {
     var neighbourPlot = new NeighbourPlot(sampleData, '#neighbourplot');
     var filter = new Filter(sampleData, neighbourPlot);
 
+    // attach behaviour to backwards and forwards buttons, unhide them
+    $backButton.unbind('click');
+    $forwardButton.unbind('click');
+
+    $backButton.on('click', function() {
+        var backId = history.back();
+        if(backId) {
+            lineplot.getSampleData(backId);
+            neighbourPlot.updatePoint(backId);
+            neighbourImages.getImages(backId);
```

Comment:
I plan on completely re-factoring the main ``script.js`` file soon.. I'll be able to address this in that PR :)